actix-rt/src/system.rs

```diff
@@ -175,20 +177,22 @@ impl System {
     ///
     /// let rest_operations = run_application();
     /// System::attach_to_tokio("actix-main-system", runtime, rest_operations);
+    /// # }
+    /// # #[cfg(not(feature = "tokio-executor"))] fn main() {} // Provie a blanket main function, so test will compile.
     /// ```
     pub fn attach_to_tokio<Fut, R>(
         name: impl Into<String>,
-        mut runtime: tokio::runtime::Runtime,
+        mut runtime: crate::executor::Runtime,
         rest_operations: Fut,
     ) -> R
     where
-        Fut: std::future::Future<Output = R>,
+        Fut: std::future::Future<Output = R> + 'static,
```

Comment:
Is adding ` + 'static` a breaking change to the API?  Is it required for both executors?

Reply:
In fact it seems that neither of them required `'static`. Removed.